slack-base/src/main/java/com/hubspot/slack/client/methods/params/chat/MessageParams.java

```diff
@@ -5,9 +5,12 @@
 
 import com.hubspot.slack.client.methods.interceptor.HasChannel;
 import com.hubspot.slack.client.models.Attachment;
+import com.hubspot.slack.client.models.blocks.Block;
 
 public interface MessageParams extends HasChannel {
   Optional<String> getText();
 
   List<Attachment> getAttachments();
+
+  List<Block> getBlocks();
```

Comment:
Will this not break implementations of this public interface?

Reply:
The only implementations we have of this interface are the immutable POJOs (which will be regenerated).  I wasn't able to find anyone directly extending this interface on github.

Reply:
Fair enough so. I was thinking it might be safer to define it as

```java
default  List<Block> getBlocks() {
 return Collections.emptyList();
}
```

Reply:
I don't think that's necessary. `List` isn't a breaking change on an immutable (it defaults to the empty list), so this method is actually unnecessary and just clutters the definition